### PR TITLE
extend CwmIntegrationTest to xpu

### PR DESCRIPTION
**Pull request by sywangyi**
@ydshieh please help review.

### CODE REVIEW

**Comment by sywangyi** [comment]
(None, None): is the logits get in cpu.

File: tests/models/cwm/test_modeling_cwm.py [hunk 1]
```diff
@@ -119,11 +122,21 @@ def test_cwm_integration(self):
             out = model(**inputs)
 
         # fmt: off
-        expected_logits = torch.tensor(
-            [0.5625, 2.9531, 9.1875, 0.5039, -0.3262, 2.2344, 3.0312, 1.5312, 0.5664, 1.5625, 2.7656, 3.4219, 2.0312, 2.1719, 1.5391, 2.5469, 2.8281, 1.8125, 1.7109, 1.3906, 1.0391, 0.1621, 0.4277, 0.1455, -0.1230, 0.8477, 2.2344, 5.2188, 1.2969, 1.5547, 0.8516, 0.7148],
-            dtype=torch.bfloat16,
-        ).to(model.device)
+        expected_logits = Expectations(
+            {
+                (None, None): torch.Tensor(
```
Comment (ydshieh) [inline tests/models/cwm/test_modeling_cwm.py]: hi, what is this value from and for?

File: tests/models/cwm/test_modeling_cwm.py [hunk 2]
```diff
@@ -157,11 +171,21 @@ def test_cwm_sliding_window_long_sequence(self):
             out = model(**inputs)
 
         # fmt: off
-        expected_logits = torch.tensor(
-            [5.2812, 6.4688, 12.8125, 4.6875, 5.2500, 4.2500, 6.9688, 4.9375, 2.7656, 6.5938, 4.9688, 1.1016, 5.9375, 3.7500, 3.1094, 5.5312, 6.1250, 4.7500, 4.5312, 2.8281, 4.0625, 3.3125, 3.9219, 3.3906, 3.1406, 3.6719, 3.2031, 7.0938, 4.8750, 6.0000, 2.7188, 6.2500],
-            dtype=torch.bfloat16,
-        ).to(model.device)
+        expected_logits = Expectations(
+            {
+                (None, None): torch.Tensor(
```
Comment (ydshieh) [inline tests/models/cwm/test_modeling_cwm.py]: same question here

File: tests/models/cwm/test_modeling_cwm.py (hunk 1 above)
Comment (sywangyi) [reply tests/models/cwm/test_modeling_cwm.py]: I run in cpu-only device, and get this value.

**Comment by yao-matrix** [comment]
@sywangyi , you need run `make style`

File: tests/models/cwm/test_modeling_cwm.py (hunk 1 above)
Comment (ydshieh) [reply tests/models/cwm/test_modeling_cwm.py]: We can remove this value obtained from CPU .

**Comment by yao-matrix** [comment]
@ydshieh , could you help take another look? Thx very much.

**Review by ydshieh — APPROVED** [review]
Thank you for the iteration and patience 🙏
